kubemarine/resources/configurations/compatibility/kubernetes_versions.yaml

```diff
@@ -131,9 +131,15 @@ compatibility_map:
     kubernetes-dashboard: v2.7.0
     local-path-provisioner: v0.0.26
     crictl: v1.29.0
+  v1.28.7:
```

Comment:
How to about to adopt latest 1.28.8 ? https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md#changelog-since-v1287
@ilia1243 @Imadzuma

Reply:
Ok